.github/workflows/handle-stale-contributor-branch-daily.yml

```
@@ -0,0 +1,29 @@
+name: Check contributor packs
+on:
+  schedule:
+    # runs at minute 0 of the 0th and 0th hour every day.
+    - cron:  '0 0 * * *'
+jobs:
+  handle_contributor_stale_branches:
+    runs-on: ubuntu-latest
+    if: github.repository == 'demisto/content'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+      - name: Install Python Dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install pipenv
+      - name: Sync Base Branch to Master
+        env:
+          CONTENTBOT_GH_ADMIN_TOKEN: ${{ secrets.CONTENTBOT_GH_ADMIN_TOKEN }}
+        run: |
+          echo "Handling stale contribution base branches (contrib/*)"
+          cd Utils/github_workflow_scripts
+          pipenv install --dev
+          pipenv run ./close_stale_contrib_branches.py
```

why do you need `cd`? any special reason?
```suggestion
          pipenv install --dev
          pipenv run Utils/github_workflow_scripts/close_stale_contrib_branches.py
```

---

no real reason

---

If I'm not mistaken, `pipenv` determines which virtual environment to use based on the directory from which it is executed. So if you run it this way, I don't think it'll locate the virtual environment specified by the `Pipfile` and `Pipfile.lock` in `Utils/github_workflow_scripts`. I imagine there is an environment variable you could set to get around this, but otherwise, I don't think this will work as you expect.